cursorless-talon/src/actions/get_text.py

```diff
@@ -1,24 +1,48 @@
-from typing import Optional
-
-from talon import actions
+from talon import Module, actions
 
 from ..targets.target_types import CursorlessTarget
 
+mod = Module()
+
+
+@mod.action_class
+class Actions:
+    def cursorless_get_text(
+        target: CursorlessTarget,
+        hide_decorations: bool = False,
```

Comment:
Not quite in love with this name. `no_decorations`? `silent`? `hide` feels a bit to active for what is really the absence of acting. Ie I might expect it to actually hide something

Reply:
I had a similar thought. `no_decorations` works for me.

Reply:
done